scripts/tpu_vm_setup.sh

```diff
@@ -9,13 +9,15 @@ sudo apt-get update && sudo apt-get install -y \
     nodejs \
     bmon \
     p7zip-full \
-    nfs-common
+    nfs-common \
+    golang
```

Comment:
Where does the golang requirement come from?

Reply:
It is a dependency of jax-smi, which I'm using to trace the TPU usage. Does this look OK?

Reply:
no worries, just making sure a random dependency hadn't snuck in :) having jax-smi included by default is a good idea.